src/butil/memory/scoped_vector.h

```diff
@@ -38,6 +41,9 @@ class ScopedVector {
   ScopedVector() {}
   ~ScopedVector() { clear(); }
   ScopedVector(RValue other) { swap(*other.object); }
+#if __cplusplus >= 201103L  // < C++11
```

Comment:
>= c++11 
  or just delete this comment since the statement is self-descriptive enough.

Reply:
fix the comment